snowplow/iglu-client-embedded/schemas/com.metabase/instance_stats/jsonschema/1-0-1

```diff
@@ -0,0 +1,212 @@
+        },
+        "required": ["key", "value"]
+      }
+    },
+    "metadata": {
+      "description": "Metadata about the anonymous stats collection",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A single metadata key/value",
+        "properties": {
+          "key": {
+            "description": "The key for this metadata",
+            "type": "string",
+            "maxLength": 255
+          },
+          "value": {
+            "description": "The value of this metadata",
+            "type": ["string", "boolean", "integer", "null"],
+            "maxLength": 255,
+            "minimum": 0,
+            "maximum": 2147483647
+          }
+        },
+        "required": ["key", "value"]
+      }
+    },
+    "metrics": {
+      "description": "Key-value pairs of metrics, with tags.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A single metric attribute",
+        "properties": {
+          "name": {
+            "description": "The unique name of the metric",
+            "type": "string",
+            "maxLength": 255
+          },
+          "value": {
+            "type": "integer",
+            "description": "The value of the metric",
+            "minimum": 0,
+            "maximum": 2147483647
+          },
+          "tags": {
+            "type": "array",
+            "description": "Tags that can be used for flagging teams / features the metric belongs to",
+            "items": {
+              "description": "a tag",
+              "type": "string",
+              "maxLength": 255
+            }
+          }
+        },
+        "required": ["name", "value", "tags"]
+      }
+    },
+    "settings": {
+      "type": "array",
+      "description": "Key-value pairs of settings, with tags.",
+      "items": {
+        "type": "object",
+        "description": "A single setting attribute",
+        "properties": {
+          "key": {
+            "type": "string",
+            "maxLength": 255,
+            "description": "The unique name of the setting"
+          },
+          "value": {
+            "type": ["string", "boolean", "integer", "null"],
+            "description": "The value of this setting",
+            "maxLength": 255,
+            "minimum": 0,
+            "maximum": 2147483647
+          },
+          "tags": {
+            "type": "array",
+            "description": "Tags that can be used for flagging teams / features the setting belongs to",
+            "items": {
+              "description": "a tag for the setting to help categorize it",
+              "type": "string",
+              "maxLength": 255
+            }
+          }
+        },
+        "required": ["key", "value", "tags"]
+      }
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "analytics_uuid",
+    "features",
+    "grouped_metrics",
+    "instance_attributes",
+    "metadata",
+    "metrics",
+    "settings"
```

Comment:
Based on the discussion with Joao (making this 2-0-0), we should be safe keeping them required. We should just make sure to send empty arrays for the attributes where we don't have the logic implemented yet.

Reply:
done.